torch/quantization/quantize.py

```diff
@@ -220,6 +226,16 @@ def quantize_qat(model, run_fn, run_args, qconfig_dict=None):
     convert(model)
     return model
 
+def quantize_dynamic(model, run_fn=None, run_args=None, qconfig_dict=None):
```

Comment:
em... why do we even need 'run' function for dynamic quantization?! the only observers you need are the weight observers and those don't need any training data

Reply:
Right. Will update it and remove the run_fn and run_args arguments.

Reply:
Updated.